EIPS/eip-1.md

```diff
@@ -168,37 +246,45 @@ if neither the email address nor the GitHub username are given.
 
 At least one author must use a GitHub username, in order to get notified on change requests and have the capability to approve or reject them.
 
-### `discussions-to` header
+#### `discussions-to` header
 
 While an EIP is a draft, a `discussions-to` header will indicate the URL where the EIP is being discussed.
 
 The preferred discussion URL is a topic on [Ethereum Magicians](https://ethereum-magicians.org/). The URL cannot point to Github pull requests, any URL which is ephemeral, and any URL which can get locked over time (i.e. Reddit topics).
 
-### `type` header
+#### `type` header
 
 The `type` header specifies the type of EIP: Standards Track, Meta, or Informational. If the track is Standards please include the subcategory (core, networking, interface, or ERC).
 
-### `category` header
+#### `category` header
 
 The `category` header specifies the EIP's category. This is required for standards-track EIPs only.
 
-### `created` header
+#### `created` header
 
 The `created` header records the date that the EIP was assigned a number. Both headers should be in yyyy-mm-dd format, e.g. 2001-08-14.
 
-### `requires` header
+#### `requires` header
 
 EIPs may have a `requires` header, indicating the EIP numbers that this EIP depends on. If such a dependency exists, this field is required.
 
 A `requires` dependency is created when the current EIP cannot be understood or implemented without a concept or technical element from another EIP. Merely mentioning another EIP does not necessarily create such a dependency.
 
-## Linking to External Resources
+### Linking to External Resources
 
 Other than the specific exceptions listed below, links to external resources **SHOULD NOT** be included. External resources may disappear, move, or change unexpectedly.
 
 The process governing permitted external resources is described in [EIP-5757](./eip-5757.md).
 
-### Execution Client Specifications
+External resources not so permitted **MAY** be included *at the Editors' discretion*.
+
```

Comment:
This is already permitted by the "**SHOULD NOT**" above.

```suggestion
```

Reply:
Hmm.  I think the "MAY" mostly eases the tone of the "SHOULD NOT", so not completely redundant.